VocaDbWeb.Core/Code/Web/AspNetCoreHttpContext.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Claims;
+using System.Security.Principal;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using VocaDb.Model.Domain.Web;
+
+namespace VocaDb.Web {
+
+	public class AspNetCoreHttpContext : IHttpContext, IServerPathMapper {
+
+		private readonly HttpContext context;

```

Comment:
Hmm, how about using IHttpContextAccessor instead? I believe that is more recommended.